pybossa/view/projects.py

```diff
@@ -3215,8 +3215,11 @@ def add_coowner(short_name, user_name=None):
         if user.id in project.owners_ids:
             flash(gettext('User is already an owner'), 'warning')
         else:
+            old_list = project.owners_ids.copy()
             project.owners_ids.append(user.id)
             project_repo.update(project)
+            auditlogger.log_event(project, current_user, 'update',
```

Comment:
on a second thought, due to limited real estate on audit page, audit page may not help identifying which user got added as coowner. would it make sense to specify which user got added as coowner? Ex
```python
auditlogger.log_event(project, current_user, 'add', 'project.coowners', "-", user.id)
```
If you agree, same update to apply from wizard code flow. Thanks!

Reply:
In the aim of keeping things simple, recording the old_list and new_list would be the most straight-forward way (the HTML will automatically trim the display). See also the screenshot in https://github.com/bloomberg/pybossa/pull/758

Otherwise, the wizard coowners endpoint would need to calculate a difference between the two lists (old_list and new_list) in order to identify added and removed users (_which it could be both adding and removing_).

Reply:
sounds good. thanks!